src/fs/hardlinkDir.ts

```diff
@@ -9,8 +9,15 @@ export default async function hardlinkDir(existingDir: string, newDir: string) {
     dirs
       .map(async (relativePath: string) => {
         const existingPath = path.join(existingDir, relativePath)
-        const newPath = path.join(newDir, relativePath);
-        const stat = await fs.stat(existingPath)
+        const newPath = path.join(newDir, relativePath)
```

Comment:
I removed the `;` here, can bring it back but that seems more consistent w/o it.